packages/api-specification/interface.ts

```diff
@@ -0,0 +1,323 @@
+declare namespace ssf {
```

Comment:
Will this work if the API isn't accessed from global scope (on the `ssf` global)? I.e. through an import?